hci_cyw43439.go

```diff
@@ -0,0 +1,27 @@
+//go:build cyw43439
+
+package bluetooth
+
+const (
+	ogfVendor = 0x3f
+
+	ocfSetBTMACAddr = 0x0001
+)
+
+func (a *Adapter) SetBdAddr(address Address) error {
+	return a.hci.setBdAddr(address)
+}
+
+func (h *hci) setBdAddr(address Address) error {
+	hciPacket := make([]byte, len(address.MACAddress.MAC))
+	// Reverse the byte order as per spec
+	for i := range address.MACAddress.MAC {
+		hciPacket[i] = address.MACAddress.MAC[len(address.MACAddress.MAC)-1-i]
+	}
+
+	if err := h.sendWithoutResponse(ogfVendor<<ogfCommandPos|ocfSetBTMACAddr, hciPacket); err != nil {
```

Comment:
Only works for this board. The opcode is vendor-specific

Reply:
Then this may belong in cyw43439 repo- given the hci packet header is a constant value and need not full hci logic to build. If the logic in this package is needed then this function may belong in an example directory under cyw43439 that uses this repo. In any case I'd avoid adding this exported API until its clearer it belongs here.

Reply:
#229 sets the MAC address using a standard opcode - does that work on CYW43439?

Reply:
I tried that first! didn't seem to work for me. I have two boards and neither worked with that PR, which is why I looked into the spec to see what was recommended, which is how I arrived at this